### PR TITLE
feat:emoji

### PR DESCRIPTION
**Pull request by adazzzz**

### CODE REVIEW

File: src/events/messageCreate.ts [hunk 1]
```diff
@@ -13,67 +14,51 @@ const mentionBot = (message: Message): Boolean => {
...
-                    : [];
+            collectNote.search(/,|，/) > 0 ||
+            collectNote.search(/\/|、/) > 0
+            ? collectNote.split(/,|，/).pop().split(/\/|、/)
+            : [];
             const discordUrl = repliedMessage.url;
-
+            
             let response = "";
+            console.log(handlerConfig.useNotion)
```
Comment (Atlasoin) [inline src/events/messageCreate.ts]: Please remove the unused console.log in all code.

File: src/events/messageReactionAdd.ts [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    
+    // reaction完成建议内容的投喂:
+    // 1.Atlas发了消息，ada@cori建议投喂
+    // 2.Cori发出一条消息@atlas
+    // 3.Atlas 点这条Cori消息。
+    // // 监控到emoji:white_check_mark: 
+    if (reaction.emoji.name === '✅'){
+        const reactionMessage = await reaction.message.fetch();
+        //检查作者是cori btw:cori id <@1018873453610274877> dino 950670085431918612 atlas 815043799322460242  ada 340572879638757386
+        if (reaction.message.author.id === '1018873453610274877' ){
```
Comment (Atlasoin) [inline src/events/messageReactionAdd.ts]: Please do not hardcode bot id here. Use environment variable instead. Reference implementation: https://github.com/DDAO-community/Cori/blob/v0.3/src/events/messageCreate.ts#L12

File: src/events/messageReactionAdd.ts [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+            //检查消息包括特定的内容.
+            if (reactionMessage.content.includes('觉得你说的很好，想让你投喂给我') ) {
+                const repliedMessage = await reaction.message.fetchReference();
+                //emoji点的人 = Cori引用消息的作者
+                if (repliedMessage.author.id === user.id){
+                    //调用投喂功能，完成投喂
+                    storage(repliedMessage,reactionMessage)
```
Comment (Atlasoin) [inline src/events/messageReactionAdd.ts]: The second param is supposed to be the message containing the title and tags. ``reactionMessage`` is the suggestion wordings sent from Cori, which is not what we want.

File: src/events/messageCreate.ts [hunk 4]
```diff
@@ -13,67 +14,51 @@ const mentionBot = (message: Message): Boolean => {
...
-            !mentionBot(message)
-        ) {
-            return;
-        }
-        // Only author could ask the bot to handle his/her message
-        if (message.author.id != repliedMessage.author.id) {
-            repliedMessage.reply(
-                `${message.author}觉得你说的很好，想让你投喂给我`
+//把存储notion和上链功能包起来，方便调用.
+export async function storage(repliedMessage,message){
```
Comment (Atlasoin) [inline src/events/messageCreate.ts]: It is suggested to use a verb as the method name. E.g. ``handle()``

**Review by Atlasoin — CHANGES_REQUESTED** [review]

File: src/events/messageReactionAdd.ts (hunk 3 above)
Comment (adazzzz) [reply src/events/messageReactionAdd.ts]: It has been improved in the latest commit.

**Review by Atlasoin — APPROVED** [review]

**Pull request merged**